content/zh/docs/ops/deployment/vm-architecture/index.md

```diff
@@ -22,7 +22,7 @@ Istio 对虚拟机的支持允许将 Kubernetes 集群外的工作负载接入
 
 {{< tabset category-name="network-mode" >}}
 
-{{< tab name="Single-Network" category-value="single" >}}
+{{< tab name="单网络" category-value="single" >}}
```

Comment:
This file has 3 SVGs:

- single-network.svg
- multi-network.svg
- service-selector.svg

Is any special reason to localize an irrelevant diagram like `discovery.svg`?

Reply:
no

Reply:
Well, would you like to localize those SVGs included in this markdown file?

- single-network.svg
- multi-network.svg

Seems the previous two diagram can be localized.

Reply:
I think I'd like to do it, but it will take some time

Reply:
All right. Not hurry. You can refer to [SVG localization guide](https://kubernetes.io/zh-cn/docs/contribute/localization/#localize-svg-images).

Reply:
Okay, thanks for the direction